src/main/index.js

```diff
@@ -1026,7 +1026,16 @@ function runApp() {
   }
 
   function baseUrl(arg) {
-    return arg.replace('freetube://', '')
+    let newArg = arg.replace('freetube://', '')
+    // add support for authority free url
+      .replace('freetube:', '')
+
+    // fix for Qt URL, like `freetube://https//www.youtube.com/watch?v=...`
+    // For details see https://github.com/FreeTubeApp/FreeTube/pull/3119
+    if (newArg.startsWith('https') && !newArg.charAt(5) !== ':') {
```

Comment:
`!newArg.charAt(5) !== ':'`
Is there something wrong here?

Reply:
Good catch